pkg/provisioning/config/parser.go

```diff
@@ -12,8 +12,6 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//go:generate mockgen -destination=mock/parser.go -package=mock -mock_names=Parser=Parser . Parser
-
```

Comment:
@lovromazgon I recall that you wanted to remove this in another PR. I've removed it here because it became a bit annoying seeing the generated but unused mock from time to time.:)

Reply:
Sure, it will also be removed in https://github.com/ConduitIO/conduit/pull/968 but I don't expect this to create a conflict.